index.html

```diff
@@ -1,31 +1,42 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- CSS STYLE SHEET -->
     <link rel="stylesheet" href="style.css">
-    <title>Portfolio</title>
+    <title>My Portfolio</title>
 </head>
+
 <body>
```

Comment:
- [ ] Well done on your mobile version, as the gaps among the sections look good visually. However, in the desktop version, the gaps do not match the design specifications. It's important to ensure consistency across different screen sizes. Please review the design carefully and update your code to match the specified gaps among the sections in the desktop version. Pay attention to the spacing and alignment to create a cohesive and visually pleasing layout. Keep up the good work! 👍✨

![Frame 3 (6)](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/55840999/d2021f7a-8104-4fb6-976a-e9aac132d92f)